region/maridia/outer.json

```diff
@@ -1033,6 +1033,24 @@
         {
           "from": 3,
           "to": [
+            {
+              "id": 4,
+              "strats": [
+                {
+                  "name": "G-Mode Morph IBJ",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [1,2,3],
+                      "mode": "any",
+                      "artificialMorph": true
+                    }},
+                    "h_canArtificialMorphIBJ",
```

Comment:
The fish+crab below the missiles are awkward to get by or slow to kill.  The rest is fine.